templates/services/merge/execution/erigon.tmpl

```diff
@@ -27,7 +27,8 @@
     - --http.addr=0.0.0.0
     - --http.port={{.ElApiPort}}
     - --txpool.disable
-    - --chain={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}
+    - --chain={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}{{if .ECBootnodes}}
```

Comment:
Research if erigon allow you to specify a custom genesis file

Reply:
Similar to Geth in that aspect. The workflow to use a custom testnet doesn't match with the Sedge uses.